src/ci/init_repo.sh

```diff
@@ -47,6 +47,7 @@ function fetch_github_commit_archive {
 }
 
 included="src/llvm-project src/llvm-emscripten src/doc/book src/doc/rust-by-example"
+included="${included} src/tools/rustfmt"
```

Comment:
Oh we probably want to back this out since it might mess up the version string of rustfmt, can tweak this later if necessary